src/commands/apextests/list.ts

```diff
@@ -20,15 +23,96 @@ export default class ApextestsList extends SfCommand<ApextestsListResult> {
       char: 'd',
       required: false,
     }),
+    format: Flags.string({
+      summary: messages.getMessage('flags.format.summary'),
+      description: messages.getMessage('flags.format.description'),
+      char: 'f',
+      required: false,
+    }),
+    // TODO: add manifest flag
   };
 
+  private static parseTestsNames(data: string): string[] {
+    // remove the prefix @Tests or @TestSuites
+    return data
+      .split(',')
+      .map((line) => line.replace(/(@Tests|@TestSuites)/, ''))
+      .map((line) => line.replace(':', ''))
+      .map((line) => line.trim())
+      .filter((line) => line.trim().length > 0);
```

Comment:
**remarks:** `.length > 0` is not necessary here

Reply:
Ooops, Copilot leak. :P